load.php

```diff
@@ -137,6 +137,41 @@ function( $module_settings ) {
 	return $modules;
 }
 
+/**
+ * Gets the active and valid performance modules.
+ *
+ * @since n.e.x.t
+ *
+ * @return array List of active valid module slugs.
+ */
+function perflab_get_active_and_valid_modules() {
+	$active_modules = perflab_get_active_modules();
+
+	if ( ! $active_modules ) {
+		return array();
+	}
+
+	$active_and_valid_modules = array();
+	foreach ( $active_modules as $module ) {
+
+		// Do not load module if it cannot be loaded, e.g. if it was already merged and is available in WordPress core.
+		$can_load_module = perflab_can_load_module( $module );
+		if ( ! $can_load_module ) {
+			continue;
+		}
+
+		// Do not load module if no longer exists.
+		$module_file = plugin_dir_path( __FILE__ ) . 'modules/' . $module . '/load.php';
+		if ( ! file_exists( $module_file ) ) {
+			continue;
+		}
```

Comment:
I agree. Before it can be loaded, we should check if the module exists or not. It's minor, but it helps for faster execution. It was applied in PR. Thanks!